### DISCUSSION TITLE
Time delay for motion detection sensors

### DISCUSSION THREAD

**Discussion by HAOccassionalUser** [blakeblackshear/frigate#1731] [Ask A Question]
HI Experts
First of all, what a fantastic piece of software. Many thanks blakeblackshear.  I have been wandering in the desert for 2 years from one NVR to the other till i stumbled upon this one just by chance. Its pretty much slotted in seamlessly into my network and needs.
 
I do have  a question. I have searched high and low but couldnt find an answer to this issue that I have. I am tracking person and car from two cameras. I have the custom add on installed, which gives me the sensors (binary_sensor.front_door_person_motion and binary_sensor.front_door_car_motion etc.). I use these to trigger flows to my appleTV. 

Whilst this works perfectly, the issue I have is that it fires multiple times and creates many alerts. So, by the time a person parks the car and walks to the front door I have about 10 alerts each.  

Is there anyway to delay the triggering of these sensors via configuration either in frigate.yml or config.yaml? 

I am not using any other automation as the homekit integration sends alerts to my appleTV (therefore TV) and phone and I get to see live pictures without having to do anything at all. If possible, I would like to avoid writing automations or sensors ( in an attempt to keep the numbers of these manageable). 

Thanks in advance
Tom

**Comment by zoltanszalontay** [comment]
I catch the events with Node-RED and consolidate corresponding alerts into one using flow context variables.

**Reply by hagak** [reply]
This needs to be handled by your automation, NodeRed is great for this.  Several ways to handle this.  The Delay Node in Rate Limit configuration works well, probably could use the trigger node as well to do this.

**Reply by NickM-27** [reply]
Also, starting in `0.11` there is an actual motion sensor for each camera with a configurable on/off delay along with the occupancy of objects which is instant.

**Comment by NickM-27** [comment]
A true motion sensor with on/off delay has been added in `0.11`. The occupancy sensors do not have delay.
